### DISCUSSION TITLE
[feature request] show link to my fork of a repo, while trying to refork it

### DISCUSSION THREAD

**Discussion by aerosayan** [community/community#14999] [Other Feature Feedback, Questions, & Ideas]
Hello,

Not sure if this is the right place to ask this ...

![fork](https://user-images.githubusercontent.com/39849461/163567851-7c1d795e-9240-4a34-bcaf-c8b07d7a6d14.png)

If I've already forked a repo and after coming back to it after a few months I tried to refork it, the ui says that no more forks can be created. This was a lot confusing as I thought github was now preventing us from forking repos for some reason.

That is, the message is not user friendly and quite confusing.

Requesting to add a link to my fork along with the message, or just redirect to my fork.

Previously github would automatically redirect to my previous fork. It was a better user experience.

Thanks

**Comment by stof** [comment]
When being a member of some organizations, the previous UI was also showing the existing forks above the list of organizations in which we could create a fork, which was handy to navigate to my fork. Having to find in the network tab is much less usable.

**Reply by aerosayan** [reply]
True. It might also be better for user experience if the message was changed to something like "You have already forked this repository. LINK_TO_YOUR_FORK".  As I mentioned before, current message of "No more forks can be created" seems very confusing, and I thought github was preventing us from creating forks for some reason.

**Comment by aerosayan** [comment]
The issue has been resolved. The link to our fork is included now. It's good for user experience. Thanks!

**Reply by stof** [reply]
When I have other organizations in which I could create a fork, this link is not available. So this is not fully resolved for me.
